pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -654,20 +682,22 @@ func selectVictimsOnNode(
 	}
 	for _, p := range violatingVictims {
 		if fits, err := reprievePod(p); err != nil {
-			klog.Warningf("Failed to reprieve pod %q: %v", p.Name, err)
-			return nil, 0, false
+			message := fmt.Sprintf("Failed to reprieve violating pod %q: %v", p.Name, err)
+			finalStatus = framework.NewStatus(framework.Error, message)
+			return nil, 0, finalStatus
```

Comment:
previous logic is that if there is any error happen within `reprievePod`,  the bool `fits` that is returned is `false`, so I follow this logic to handle the `status` as well, but I think this is debatable, because we already found the candidate, the error we found  for such `reprieve` should not de-candidate this node. 

If you agree with my points here, I'd happy to post a follow up to add it.

Reply:
lets keep it as is, this is an unexpected error, something really is off if this happens.

Reply:
ACK.